sonic_installer/main.py

```diff
@@ -579,11 +579,16 @@ def install(url, force, skip_platform_check=False, skip_migration=False, skip_pa
 
         # Calling verification script by default - signature will be checked if enabled in bios
         echo_and_log("Verifing image {} signature...".format(binary_image_version))
-        if not bootloader.verify_image_sign(image_path):
-            echo_and_log('Error: Failed verify image signature', LOG_ERR)
-            raise click.Abort()
-        else:
-            echo_and_log('Verification successful')
+        try:
+            if not bootloader.verify_image_sign(image_path):
+                echo_and_log('Error: Failed verify image signature', LOG_ERR)
+                raise click.Abort()
+            else:
+                echo_and_log('Verification successful')
+        except AttributeError:
```

Comment:
>except AttributeError: [](http://example.com/codeflow?start=8&length=22)

This looks like a hack to workaround the logic error inside implementation.

Reply:
@qiluo-msft whats in your mind? How do you want the fix ?